src/users/components/invite-user-permissions/invite-user-permission.component.html

```diff
@@ -1,31 +1,47 @@
   <!--permissions -->
   <ng-container [formGroup]="inviteUserForm">
-  <lib-gov-uk-form-group-wrapper
-    [config]="{hint: 'Choose what the user will be able to do. You can change this later.', legend: 'Permissions', key: 'roles'}"
-    [formGroupName]="'roles'"
-    [error]="errorMessages">
+    <lib-gov-uk-form-group-wrapper
+      [config]="{hint: 'Choose at least one action the user can do. You can change this later.', legend: 'Permissions', key: 'roles'}"
+      [formGroupName]="'roles'"
+      [error]="errorMessages">
 
-    <div class="govuk-checkboxes">
-      <lib-gov-checkbox
-        [group]="inviteUserForm.controls['roles']"
-        [isChecked]="isPuiCaseManager"
-        [config]="{value: 'pui-case-manager',  label: 'Manage Cases', name: 'roles', hint: 'View, create and progress cases.', focusOn: 'roles'}">
-      </lib-gov-checkbox>
+      <div class="govuk-checkboxes">
+        <lib-gov-checkbox
+          [group]="inviteUserForm.controls['roles']"
+          [isChecked]="isPuiCaseManager"
+          [config]="{value: 'pui-case-manager',  label: 'Manage cases', name: 'roles', hint: 'View, create and progress cases.', focusOn: 'roles'}">
+        </lib-gov-checkbox>
 
-      <lib-gov-checkbox
-        [group]="inviteUserForm.controls['roles']"
-        [isChecked]="isPuiUserManager"
-        [config]="{value: 'pui-user-manager', label: 'Manage Users', name: 'roles',
-        hint: 'Invite users.'}">
-      </lib-gov-checkbox>
+        <lib-gov-checkbox
+          [group]="inviteUserForm.controls['roles']"
+          [isChecked]="isPuiUserManager"
+          [config]="{value: 'pui-user-manager', label: 'Manage users', name: 'roles',
+          hint: 'Invite users.'}">
+        </lib-gov-checkbox>
 
-      <lib-gov-checkbox
-        [group]="inviteUserForm.controls['roles']"
-        [isChecked]="isPuiOrganisationManager"
-        [config]="{value: 'pui-organisation-manager', label: 'Manage Organisation', name: 'roles',
-        hint: 'View organisation name and addresses.'}">
+        <lib-gov-checkbox
+          [group]="inviteUserForm.controls['roles']"
+          [isChecked]="isPuiOrganisationManager"
+          [config]="{value: 'pui-organisation-manager', label: 'Manage organisation', name: 'roles',
+          hint: 'View the organisation name and addresses.'}">
+        </lib-gov-checkbox>
+
+        <lib-gov-checkbox
```

Comment:
this seems like not synchronized from master, I can't see the latest change I made in another PR:
https://github.com/hmcts/rpx-xui-manage-organisations/pull/623
Could you double check what/how do you synchronize from master?